crates/cast/bin/cmd/send.rs

```diff
@@ -59,6 +59,10 @@ pub struct SendTxArgs {
 
     #[command(flatten)]
     eth: EthereumOpts,
+
+    /// The path of blob data to be sent.
+    #[arg(long, value_name = "BLOB_DATA_PATH", requires = "blob")]
```

Comment:
can also use `conflicts_with` to avoid doing the `legacy` check down below: https://docs.rs/clap/latest/clap/struct.ArgGroup.html#method.conflicts_with